subjects/invert_sentence/README.md

```diff
@@ -30,7 +32,7 @@ And its output:
 ```console
 $ cargo run | cat -e
 Awesome is Rust$
-word2 word1$
+   word2     word1  $
```

Comment:
Here it seems like the spaces are wrong... the first one should be 2 spaces and not 3 at the start and 3 and not 2 at the end:
`  word2     word1   `

Reply:
indeed